sdk/cosmosdb/cosmos/writeSDKVersion.js

```diff
@@ -1,6 +1,6 @@
 const fs = require("fs");
 const version = require("./package.json").version;
-const fileName = "./dist-esm/src/common/constants.js";
+const fileName = "./src/common/constants.ts";
```

Comment:
This seems like it's going to write the version number into the TypeScript sources instead of into the compiled sources. This isn't safe to change.